app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,10 @@
+class LikesController < ApplicationController
+  def new
+    post = Post.find(params[:post_id])
+    author = User.find(params[:user_id])
+    like = Like.new(author:, post:)
+    return unless like.save
+
+    redirect_to user_posts_path
+  end
+end
```

Comment:
- [ ] [OPTIONAL] it would be better if you use the [strong parameters](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides) to make your app more safer